snap/local/runtime-helpers/bin/mongo-worker-wrapper.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+
+
+while true; do
```

Comment:
Hmmmm, is this identical to what's in delhi? Shouldn't there be a retry limit so that this doesn't run forever on a mis-configured system? I'm fine if you want to file an issue for this.

Reply:
Hmm it is indeed the same... I will file an issue for this.

Reply:
Filed as #944